sql/_procedures.sql

```diff
@@ -46,20 +46,24 @@ BEGIN
             EXECUTE format($n$
                 UPDATE %1$I SET %2$I = %4$L::%5$s WHERE %1$I.uid = %3$L;
             $n$, _table, _field, _uid, _new_val, field_type);
+            EXECUTE format($n$
+                DELETE FROM modifications_applied WHERE uid = %1$L AND field = %2$L AND old_value = %3$L;
+            $n$, _uid, _field, _old_val, _new_val);
+            EXECUTE format($n$
```

Comment:
Now that by value modifications have no duplicates, do we even need this second delete from `modifications_not_applied`?

Reply:
I think it makes more sense for when the uid is not null, easier for mistakes to slip into the modifications table where one row has one correction and another row has another correction and we make sure the modifications not applied reflects the latest correction. But more confusing than it's worth by recoding by value as there aren't supposed to be any duplicates

Reply:
So I think by this logic it makes sense to move to within the if statement as is? Also challenging to think about all the scenarios when it is all hypothetical. I think we can refactor as needed when future we have real new modification records to deal with.